src/index.js

```diff
@@ -687,6 +691,57 @@ function transformAstro(ast, { env, changes }) {
   }
 }
 
+function transformMarko(ast, { env, changes }) {
+  const nodesToVisit = [ast]
+  while (nodesToVisit.length > 0) {
+    const currentNode = nodesToVisit.pop()
+    switch (currentNode.type) {
+      case 'File':
+        nodesToVisit.push(currentNode.program)
+        break
+      case 'Program':
+        nodesToVisit.push(...currentNode.body)
+        break
+      case 'MarkoTag':
+        nodesToVisit.push(...currentNode.attributes)
+        nodesToVisit.push(currentNode.body)
+        break
+      case 'MarkoTagBody':
+        nodesToVisit.push(...currentNode.body)
+        break
+      case 'MarkoAttribute':
+        if (currentNode.name === "class") {
+          switch (currentNode.value.type) {
+            case 'ArrayExpression':
+              const classList = currentNode.value.elements
```

Comment:
Because `classList` is an array of Nodes, we can't sort it directly using `sortClassList`. We also can't sensibly sort non-string literal Node types, so we move them to the front and sort the remaining classes.

Reply:
Hey! So I don't think we want to sort the array itself. We've always avoided doing this and only sort strings and template literals.

For instance in Vue, which has a similar syntax, we would sort:

```vue
<!-- before -->
<div class="text-gray-900 mt-2"></div>

<!-- after -->
<div class="mt-2 text-gray-900"></div>
```

But we don't sort this:

```vue
<!-- before -->
<div :class="['text-gray-900', 'mt-2']"></div>

<!-- after -->
<div :class="['text-gray-900', 'mt-2']"></div>
```

But we will sort multiple classes in each string:

```vue
<!-- before -->
<div :class="['bg-gray-900 mt-2', 'underline font-bold']">

<!-- after -->
<div :class="['mt-2 bg-gray-900', 'font-bold underline']">
```

Are you able to update this PR to this behavior? 🙏

Reply:
Oh yeah, it didn't feel right sorting the array. :)

Thanks, I've made that change and updated the tests.